app/components/chat-message.js

```diff
@@ -1,20 +1,28 @@
-import Component from '@ember/component';
+import classic from 'ember-classic-decorator';
```

Comment:
more info about classic decorator
https://github.com/emberjs/ember-classic-decorator#why-do-i-need-this-decorator